setup.cfg

```diff
@@ -1,10 +1,12 @@
 [metadata]
 name = asdf
 description = Python tools to handle ASDF files
+long_description = file: README.rst
 long_description_content_type = text/x-rst
 author = Erik Bray, Dan D'Avella, Michael Droettboom
 author_email = mdroe@stsci.edu
-license = BSD
+license = BSD-3-Clause
+license_file = LICENSE.rst
 url = http://github.com/spacetelescope/asdf
 edit_on_github = False
 github_project = spacetelescope/asdf
```

Comment:
this can be changed to `asdf-format/asdf`, and the url too.

Reply:
Yes!  Will do.